installer/install.iss

```diff
@@ -57,7 +57,7 @@ DisableDirPage=auto
 DefaultGroupName={#APP_NAME}
```

Comment:
If you see in the list of changes for this commit, the `gpl-2.0.rtf` was lifted up a directory in that very commit. It currently resides in the `build-extra` top-level directory. Did I misunderstand what you're trying to tell me to do?